frontend/src/components/Quiz/Result.jsx

```
@@ -0,0 +1,136 @@
+                      <h4 className="font-medium">
+                        Question {index + 1}: {q.text.length > 50 ? `${q.text.substring(0, 50)}...` : q.text}
+                      </h4>
+                      <span className={q.is_correct ? 'text-green-600' : 'text-red-600'}>
+                        {q.is_correct ? '✓' : '✗'}
+                      </span>
+                    </div>
+                    <div 
+                      id={`question-${index}`} 
+                      className={q.is_correct ? 'hidden' : ''}
+                      role="region"
+                      aria-labelledby={`question-${index}-header`}
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<section>


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)